Makefile

```diff
@@ -1,7 +1,7 @@
-PREFIX ?=	$PREFIX
+include .config
```

Comment:
It works on NetBSD. I think all variants support that syntax... Which implementations are you testing with?

Em 6 de maio de 2016 20:22:16 GMT-03:00, Kyle Isom <notifications@github.com> escreveu:
>> @@ -1,7 +1,7 @@
>> -PREFIX ?=	$PREFIX
>> +include .config
>
>FYI: this is specific to GNU make, and won't work with BSD make.
>
>---
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/kisom/rawk/pull/8/files/34cb8c0880ed47d0857ab76632d05f00814c363e#r62403756

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.